### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.3`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/BenSampo/laravel-enum/pull/242/commits/81c5c56b6b74f7f50615577ab1a3286a19634bff?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-enum.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "bensampo/laravel-enum": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        php: [7.3, 7.4, 8.0, 8.1]
```
Comment (jasonmccreary) [inline .github/workflows/tests.yml:12]: Oops, may want to keep `8.1`, although it is marked to `exclude`.

File: composer.json [hunk 2]
```diff
@@ -21,18 +21,18 @@
     "require": {
         "php": "^7.3|^8.0",
```
Comment (BenSampo) [inline composer.json:22]: @jasonmccreary L9 drops support for PHP 7.x as far as I know - should this have been updated to `^8.0`?

**Comment by BenSampo** [comment]
Blocked by https://github.com/hanneskod/classtools

File: composer.json (hunk 2 above)
Comment (jasonmccreary) [reply composer.json:22]: Yes. Shift must have seen the original, compound constraint as satisfying the PHP 8.0 requirement. I'll look into tweaking that for the future.

**Comment by hlaporthein** [comment]
That would be great if it support Laravel 9. So, we could test it and add into our projects

**Comment by benfiratkaya** [comment]
I'm waiting

**Comment by jasonmccreary** [comment]
@benfiratkaya, there are [instructions for adding it to your project](https://github.com/BenSampo/laravel-enum/pull/242#issuecomment-1025908208) from this branch in this PR. Don't be pushy. 👎

**Comment by BenSampo** [comment]
Merged and tagged (version 5.0).

Thanks for your input on this @jasonmccreary 👍

**Pull request merged**